ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml

```diff
@@ -52,14 +52,15 @@ StatusScrollView {
     QtObject {
         id: d
 
+        property bool isAssetView: (optionsTab.currentIndex === 1)
```

Comment:
`readonly`?

Reply:
I will modify it in my [pr](https://github.com/status-im/status-desktop/pull/10774).